.github/workflows/docspublish.yml

```diff
@@ -6,13 +6,44 @@ on:
       - master
 
 jobs:
+  update-cli-screenshots:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip poetry
+          poetry --version
+          poetry install
+      - name: Update CLI screenshots
+        run: |
+          poetry run python scripts/gen_cli_help_screenshots.py
+      - name: Commit and push updated CLI screenshots
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git add docs/images/cli_help
+          git commit -m "docs(cli/screenshots) update CLI screenshots" -m "[skip ci]"
```

Comment:
```suggestion
          git commit -m "docs(cli/screenshots): update CLI screenshots" -m "[skip ci]"
```

what is this second -m for?

Reply:
Is an git syntax for declaring the commit description. First "-m" is for commit title and second "-m" for commit description like this:

![image](https://github.com/commitizen-tools/commitizen/assets/48416792/43512f7b-bf37-4317-9d65-c08786d77e84)

I don't like the "[skip ci]" on commit title so I put it on commit description.

Reply:
Got it 

```

       -m <msg>, --message=<msg>
           Use the given <msg> as the commit message. If multiple -m options are given, their values are concatenated as separate paragraphs.

           The -m option is mutually exclusive with -c, -C, and -F.

```